### ISSUE TITLE
[TOC] Show hints for unavailable layers

### ISSUE THREAD

**Issue by antoniave** [open-pioneer/trails-openlayers-base-packages#6]
As a user I want to see if a layer that is configured in the map is not available (for example because the service is offline or I have network issues in my client). I want to get a visual feedback and see a hint message that the layer cannot be used. 

Acceptance criteria: 
- If a layer is not available, the item in the toc is shown in a color with less contrast (such as grey) and a red exclamation mark.
- Functions at the affected layer item do not lead to erros when using them.
  - If possible, functions such as visible-checkbox or info button (see #5) go in a disabled state so that they cannot be clicked.
- Once the toc is visible, the whole layer list is shown immediately and not blocked by unavailable layers.

Out of scope:
- Hints for layers that are currently not available due to other reasons (such as scale dependancy).
